stack.yaml

```diff
@@ -131,3 +142,11 @@ ghc-options:
   cardano-sl:            -Werror
   cardano-sl-tools:      -Werror
   cardano-sl-lwallet:    -Werror
+  cardano-sl-explorer:   -Werror
+
+# Added for explorer
+flags:
+  cardano-sl:
+    with-explorer: True
```

Comment:
@gromakovsky And you want me to add a flag for not building _explorer_?

So do you want me to change `scripts/build/cardano-sl.sh` so it contains `explorer` build by default? If I add `--no-explorer` it should not build parts b. and c., and the flag can be set to `False`, right? And you want me to add `explorer` c. part build into the script?

- [ ] Did I get this finally?